### ISSUE TITLE
as someone interested in civic tech in south africa I'd like to join the conversation

### ISSUE THREAD

**Issue by jbothma** [schalkventer/civic-tech-za#6]
link to:

- zatech slack and interesting channels
- code4africa slack and interesting channels
- civic tech innovation network newsletter signup/archive
- hacks/hackers google group

**Comment by schalkventer** [comment]
Yes, this is a great idea @jbothma! From the feedback I received from @YaguraStation it seems like the step-by-step guide I wrote for #civic-tech on ZA Tech seemed to be helpful. It might be a good idea to create similar step-by-step guides for joining any of the above as well, since it would be nice to have as low a barrier to entry in terms of experience/digital aptitude.

**Comment by schalkventer** [comment]
@jbothma Can you provide a link to the resources above (except for the two Slack channels/groups). I'll add them to the directory from my side. 👍

**Comment by jbothma** [comment]
Civic tech innovation network
- Home http://www.journalism.co.za/civic-tech-innovation-network/
- newsletter http://us14.campaign-archive.com/home/?u=559d2c371baafd0fbe4a2dbae&id=b4f2a75524
- blog on medium https://medium.com/civictech

I'll have to confirm the existence of the google group I thought I saw

**Comment by jbothma** [comment]
Hacks/Hackers google groups
- https://groups.google.com/forum/#!forum/hackshackers-capetown
- https://groups.google.com/forum/#!forum/hackshackers-johannesburg
